packages/components/src/ResourcePicker/Resource/__snapshots__/Resource.snap.test.js.snap

```diff
@@ -0,0 +1,55 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Resource component snaps renderers should render a Resource without icon 1`] = `
+<Translate(Resource)
```

Comment:
Your snapshots are not relevant.
Can you render `Resource.WrappedComponent` instead of `Resource` ?